File: Api/Quiz.Domain/Entities/Question.cs
namespace Quiz.Domain.Entities
{
    public class Question : Entity
    {
        public Question(string title, string correctAnswer, string company, 
        string answerA, string answerB, string answerC, string answerD)
        {
            Title = title;
            CorrectAnswer = correctAnswer;
            Company = company;
            AnswerA = answerA;
            AnswerB = answerB;
            AnswerC = answerC;
            AnswerD = answerD;
        }
        public string Title { get; private set; }
        public string CorrectAnswer { get; private set; }
        public string Company { get; private set;}
        public string AnswerA { get; private set; }
        public string AnswerB { get; private set; }
        public string AnswerC { get; private set; }
        public string AnswerD { get; private set; }
        
    }
}

File: mobile/lib/bussiness/entities/question.dart
class Question {
  String id;
  String title;
  String correctAnswer;
  String answerA;
  String answerB;
  String answerC;
  String answerD;
  String company;

  Question({
    this.id,
    this.title,
    this.company,
    this.correctAnswer,
    this.answerA,
    this.answerB,
    this.answerC,
    this.answerD
  });

  Question.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    title = json['title'];
    correctAnswer = json['correctAnswer'];
    company = json['company'];
    answerA = json['answerA'];
    answerB = json['answerB'];
    answerC = json['answerC'];
    answerD = json['answerD'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> question = Map<String, dynamic>();
    question['id'] = id;
    question['title'] = title;
    question['correctAnswer'] = correctAnswer;
    question['company'] = company;
    question['answerA'] = answerA;
    question['answerB'] = answerB;
    question['answerC'] = answerC;
    question['answerD'] = answerD;
    return question;
  }
}